### PR TITLE
Remove and upload capabilities nodejs

### PR DESCRIPTION
**Pull request by abaliunov-sc**
#18 #21

### CODE REVIEW

File: client-react/src/client/components/ConfirmDialog/ConfirmDialog.less [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.set-name-dialog {
```
Comment (kvolkovich-sc) [inline client-react/src/client/components/ConfirmDialog/ConfirmDialog.less]: Remove this CSS class

File: client-react/src/client/components/ConfirmDialog/ConfirmDialog.react.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+const defaultProps = {
+  headerText: 'Do you really want to remove the file?',
+  submitButtonText: 'Ok',
+  autofocus: false,
+  onHide: () => {},
+  onSubmit: () => {}
+};
+
+export default
+class CancelOkDialog extends Component {
```
Comment (kvolkovich-sc) [inline client-react/src/client/components/ConfirmDialog/ConfirmDialog.react.js]: Rename `CancelOkDialog` to `ConfirmDialog`

File: client-react/src/client/components/ConfirmDialog/ConfirmDialog.react.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+    return (
+      <Dialog onHide={onHide}>
+        <div className="oc-fm--dialog__content" onKeyDown={this.handleKeyDown}>
+          <div className="oc-fm--dialog__header">
+            {headerText}
+          </div>
+
+          <div className="oc-fm--dialog__horizontal-group oc-fm--dialog__horizontal-group--to-right">
+            <button type="button" className="oc-fm--dialog__button oc-fm--dialog__button--default" onClick={onHide}>
```
Comment (kvolkovich-sc) [inline client-react/src/client/components/ConfirmDialog/ConfirmDialog.react.js]: Please create `handleCancel` method as `handleSubmit` done

File: client-react/src/client/components/ConfirmDialog/ConfirmDialog.react.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+const propTypes = {
+  headerText: PropTypes.string,
+  submitButtonText: PropTypes.string,
+  onHide: PropTypes.func,
+  onSubmit: PropTypes.func
+};
+const defaultProps = {
+  headerText: 'Do you really want to remove the file?',
+  submitButtonText: 'Ok',
```
Comment (kvolkovich-sc) [inline client-react/src/client/components/ConfirmDialog/ConfirmDialog.react.js]: Make `Ok` uppercase

File: client-react/src/client/connectors/nodejs_v1/capabilities/delete-resource.js [hunk 5]
```diff
@@ -17,7 +18,25 @@ function handler(apiOptions, {
...
+  let selectedResources = getSelectedResources();
+  showDialog((
+    <ConfirmDialog
+      onHide={hideDialog}
+      onSubmit={async () => {
+        hideDialog();
+        api.removeResources(apiOptions, selectedResources, { onSuccess, onFail });
+      }}
+      headerText={`Do you really want to remove\n${selectedResources.length > 1 ? `${selectedResources.length} files` : `the file ${selectedResources[0].name}`}?`}
+      submitButtonText={'Ok'}
```
Comment (kvolkovich-sc) [inline client-react/src/client/connectors/nodejs_v1/capabilities/delete-resource.js]: Make `Ok` uppercase

**Comment by abaliunov-sc** [comment]
Done

**Pull request merged**